static/app/views/onboarding/components/footer.spec.tsx

```diff
@@ -0,0 +1,143 @@
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {
+  render,
+  renderGlobalModal,
+  screen,
+  userEvent,
+  waitFor,
+} from 'sentry-test/reactTestingLibrary';
+
+import {PersistedStoreProvider} from 'sentry/stores/persistedStore';
+import * as useSessionStorage from 'sentry/utils/useSessionStorage';
+import {Footer, OnboardingStatus} from 'sentry/views/onboarding/components/footer';
+
+describe('Onboarding Footer', function () {
+  it('waiting for error ui', async function () {
+    const {project, organization, router, route} = initializeOrg();
+
+    MockApiClient.addMockResponse({
+      url: `/projects/${organization.slug}/${project.slug}/`,
+      body: project,
+    });
+
+    render(
+      <PersistedStoreProvider>
+        <Footer
+          projectSlug={project.slug}
+          location={router.location}
+          route={route}
+          router={router}
+        />
+      </PersistedStoreProvider>,
+      {
+        organization,
+      }
+    );
+
+    // Skip onboarding link
+    expect(screen.getByRole('link', {name: 'Skip Onboarding'})).toHaveAttribute(
+      'href',
+      '/organizations/org-slug/issues/?referrer=onboarding-first-event-footer-skip'
+    );
+
+    // Error status
+    expect(screen.getByText('Waiting for error')).toBeInTheDocument();
+
+    // Explore Sentry button disabled
+    expect(screen.getByRole('button', {name: 'Explore Sentry'})).toBeDisabled();
+
+    userEvent.hover(screen.getByRole('button', {name: 'Explore Sentry'}));
+
+    // Explore Sentry button tooltip
+    await waitFor(() => {
+      expect(screen.getAllByText('Waiting for error')).toHaveLength(2);
+    });
+  });
+
+  it('processing error ui', async function () {
+    const {project, organization, router, route} = initializeOrg();
+
+    // Mock useSessionStorage hook to return the mocked session data
+    jest.spyOn(useSessionStorage, 'useSessionStorage').mockImplementation(() => [
+      {
+        status: OnboardingStatus.PROCESSING,
+        firstIssueId: undefined,
+      },
+      jest.fn(),
+      jest.fn(),
+    ]);
+
+    render(
+      <PersistedStoreProvider>
+        <Footer
+          projectSlug={project.slug}
+          location={router.location}
+          route={route}
+          router={router}
+        />
+      </PersistedStoreProvider>,
+      {
+        organization,
+      }
+    );
+
+    // Skip onboarding link is gone
+    expect(screen.queryByRole('link', {name: 'Skip Onboarding'})).not.toBeInTheDocument();
+
+    // Error status
+    expect(screen.getByText('Processing error')).toBeInTheDocument();
+
+    renderGlobalModal();
+
+    userEvent.click(screen.getByRole('button', {name: 'Explore Sentry'}));
+
+    // Display are you sure modal
+    expect(await screen.findByText('Are you sure?')).toBeInTheDocument();
+  });
+
+  it('error processed ui', function () {
```

Comment:
do you want to name this step differently?

Reply:
we are calling it "Error Processed!" 

![image](https://user-images.githubusercontent.com/29228205/222670473-612f1753-c074-447b-958f-06dd3037aa24.png)


What exactly do you mean?

Reply:
you misunderstand my comment:
on line 57 you have the same name, so I was wondering if we can name it differently.

Reply:
I see... but on line 57 I call it "processing error ui"... slightly different :)

Reply:
nvm i misread it.